src/index.js

```diff
@@ -0,0 +1,230 @@
+"user strict"
+const locationURL = 'http://127.0.0.1:5000/location'
+const weatherURL = 'http://127.0.0.1:5000/weather'
```

Comment:
Nice job using constant variables to reference the endpoints.
```suggestion
const LOCATION_URL = 'http://127.0.0.1:5000/location'
const WEATHER_URL = 'http://127.0.0.1:5000/weather'
```